stripe.go

```diff
@@ -393,8 +393,29 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 			s.LeveledLogger.Errorf("Request failed with error: %v", err)
 		} else if res.StatusCode >= 400 {
 			err = s.ResponseToError(res, resBody)
-			s.LeveledLogger.Errorf("Request failed with body: %s (status: %v)",
-				string(resBody), res.StatusCode)
+
+			if stripeErr, ok := err.(*Error); ok {
```

Comment:
@brandur-stripe -- `err` is now not logged if the type assertion fails.  Not a problem if err always has type `*Error`, but might be a problem otherwise.

Reply:
Thanks. To be fair, this should basically ~never happen (only in the case where we got back some mangled error body from the Stripe API, but where it somehow managed to close the connection properly, etc.), but yeah, we should make sure that it gets logged.

Created a small tweak on the last patch in #939.